tests/e2e/core-tests/specs/shopper/front-end-checkout-login-account.test.js

```diff
@@ -37,7 +38,7 @@ const runCheckoutLoginAccountTest = () => {
 
 			// Add simple product to cart and proceed to checkout
 			await shopper.goToShop();
-			await shopper.addToCartFromShopPage(simpleProductName);
+			await shopper.addToCartFromShopPage( productId );
```

Comment:
`simpleProductName` is unused now, so we can drop the declaration on 23 now.

Reply:
Removed in https://github.com/woocommerce/woocommerce/pull/30822/commits/ef778b898599d50e3775bbdc5909146cb89e4bfa